.github/workflows/deploy.yml

```
@@ -133,13 +133,36 @@ jobs:
         run: |
           tar -czf "$ARCHIVE_NAME" src/ docs/api/openapi-generated.yaml infra/nginx/
 
-      - name: Upload archive and deploy script to server
+      - name: Render .env from per-env template
+        env:
+          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
```

actionlint (1.7.12)

[error] property "postgres_password" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; corporate_email: string; corporate_password: string; github_token: string; sentry_auth_token: string}

(expression)